docs/docusaurus.config.js

```diff
@@ -81,9 +87,8 @@ const config = {
           customCss: require.resolve("./src/scss/application.scss"),
         },
         pages: {
-          path: "src/pages",
-          include: ["**/*.md", "**/*.mdx"],
```

Comment:
My very first attempt at making this page was to make a plain html page, so I tried naively adding the `.html` extension to this array. But that didn't work and then I realized that I would need to make a React `.jsx` file. But rather than add `.jsx` to this array, I figured we might as well just use, which is [`**/*.{js,jsx,ts,tsx,md,mdx}`](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-pages#include).

Reply:
let's remove the `include` I think I only added these because I added a `.mdx` in the first instance